src/App.test.js

```diff
@@ -1,8 +1,20 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import Calculator from './components/Calculator';
+import Quote from './components/Quote';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+test('contains -William Paul Thurston', () => {
+  render(<Quote />);
+  const linkElement = screen.getByText(/-William Paul Thurston/i);
   expect(linkElement).toBeInTheDocument();
+
+  // const paragraphTag = screen.getByRole("paragraph");
+  // expect(paragraphTag).toBeInTheDocument();
+});
+
+test('Contain button element', () => {
+  render(<Calculator />);
+  const buttonElement = screen.getByRole('button', {
+    name: 'AC',
+  });
+  expect(buttonElement).toBeInTheDocument();
 });
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/220761755-bfd7e361-e218-4015-b29c-8bed9964269b.png)

- [ ] Great work on implementing the tests so far but I guess some of your tests are not passing :fearful: I apologize on behalf of the previous reviewer for not pointing out this issue :pray: It means that your application is not behaving as expected :thinking: It would be nice if you make your tests pass to ensure that your application is behaving as expected :+1: